### ISSUE TITLE
Click payload sent twice for Xiaomi wireless switch WXKG01LM

### ISSUE THREAD

**Issue by PattaFeuFeu** [Koenkk/zigbee2mqtt#1645]
# Bug Report
## What happened

When pairing a WXKG01LM Xiaomi wireless switch and using it, the payload is sent twice for each press—no matter the type. As you can see in the attached logs, whenever I pressed once, twice, three times, four times, or did a long press, first a payload with the specification of presses is sent and then another message containing just an unspecified click.

## What did you expect to happen

For the payload to be sent only once, and only with the specified click.

## How to reproduce it (minimal and precise)

Enable homeassistant in config, add sensor, use sensor

## Debug Info
zigbee2mqtt version:
CC253X firmware version:

Log: https://pastebin.com/V0hHB5Js

**Comment by Koenkk** [comment]
This is expected, duplicate of https://github.com/Koenkk/zigbee2mqtt/issues/1457#issuecomment-485401419
